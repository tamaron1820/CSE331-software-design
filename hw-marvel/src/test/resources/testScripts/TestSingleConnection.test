# Test if two nodes does not have any path

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/singleConnectionMarvel.csv

LoadGraph singleConnection singleConnectionMarvel.csv

FindPath singleConnection HUMAN-ROBOT GORILLA-MAN