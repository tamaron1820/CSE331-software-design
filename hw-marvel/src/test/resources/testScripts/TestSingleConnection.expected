# Test if two nodes does not have any path

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/singleConnectionMarvel.csv

loaded graph singleConnection

path from HUMAN-ROBOT to GORILLA-MAN:
HUMAN-ROBOT to GORILLA-MAN via WI?-9